sale_coupon_auto_refresh/models/sale_coupon_refresh_mixin.py

```diff
@@ -17,11 +17,21 @@ class SaleCouponRefreshMixin(models.AbstractModel):
 
     @api.model
     def _get_auto_refresh_coupons_triggers(self) -> set:
-        """Returns set of fields which trigger the recomputation
+        """Returns set of fields which trigger the recomputation. We can configure
+        additional triggers via config parameters setting them up as list of fields
+        separated by commas. For example, for sale.order, the config param key would be
+        sale_coupon_auto_refresh.sale_order_triggers and the value something like:
+        warehouse_id,carrier_id
 
-        Hook method to be overridden if necessary
+        The method is overriden in the proper modules to set some basic triggers though
         """
-        return set()
+        additional_triggers = (
+            self.env["ir.config_parameter"]
+            .sudo()
+            .get_param("sale_coupon_auto_refresh.%s_triggers" % (self._table), "")
+            .split(",")
```

Comment:
Maybe someone introduce the list of fields as `field1, field2, field3`

```suggestion
            .split(",").strip()
```

Reply:
Done with `.replace(" ", "").split(",")` `strip()` only removes leading and trailing spaces.

Reply:
Sorry, I tried to put the `strip()` on each element of the list.